python.md

```diff
@@ -0,0 +1,95 @@
+# Python on Graviton processors
+
+Build Python software tuned for Graviton processors.
+
+## Overview
+
+Python relies on native code to achieve high performance.  For scientific and
+numerical applications NumPy and SciPy provide an interface to high performance
+computing libraries such as ATLAS, BLAS, BLIS, OpenBLAS, etc.  These libraries
+contain code tuned for Graviton processors.
+
+## BLIS may be a faster BLAS
+
+The default SciPy and NumPy binary installations with `pip3 install numpy scipy`
+are configured to use OpenBLAS. On Ubuntu `apt install python3-numpy
+python3-scipy` will install NumPy and SciPy with BLAS and LAPACK libraries.
+The default installations of SciPy and NumPy are easy to setup and well
+tested.
+
+Some workloads will benefit from using BLIS. Benchmarking SciPy and NumPy
+workloads with BLIS might allow to identify additional performance improvement.
+If NumPy and SciPy detect the presence of the BLIS library, they will use BLIS
+in priority over the same functionality from BLAS and OpenBLAS.
+OpenBLAS or LAPACK libraries need to be installed along BLIS to provide LAPACK
+functionality.
+
+### Install BLIS on Ubuntu and Debian
+
+To install the prerequisites to build SciPy and NumPy with BLIS and OpenBLAS on
+Ubuntu and Debian:
+```
+sudo apt -y install build-essential python3-pip libopenblas-dev gfortran libblis-dev
+```
+
+To build and install NumPy and SciPy with BLIS and OpenBLAS:
+```
+git clone https://github.com/numpy/numpy/ $HOME/numpy
+cd $HOME/numpy;  pip3 install .
+
+git clone https://github.com/scipy/scipy/ $HOME/scipy
+cd $HOME/scipy;  pip3 install .
```

Comment:
should we just tell them to apt install python3-scipy or numpy then? Not sure how much we're getting with building it themselves vs update-alternatives and using the Ubuntu provided package.

Reply:
I updated the Ubuntu instructions to use alternatives.  It is much faster to install from pre-built binaries than re-building.